projects/demo-integrations/cypress/tests/addon-commerce/input-card-grouped.spec.ts

```diff
@@ -19,4 +18,85 @@ describe('InputCardGrouped', () => {
         cy.get('@wrapper').click({force: true});
         cy.matchImageSnapshot('03-input-card-grouped-unfocused', {capture: 'viewport'});
     });
+
+    it('set value and disable', () => {
+        cy.tuiVisit('components/input-card-grouped/API?tuiMode=null');
+        cy.get('#demoContent').as('wrapper');
+
+        cy.get('@wrapper')
+            .findByAutomationId('tui-input-card-grouped__card')
+            .type('1234 4567 1000 1112')
+            .wait(5000); // waiting before closing toast for skip flaky test
+
+        cy.get('.t-table tr')
+            .eq(1) // formControl.disable()
+            .findByAutomationId('tui-toggle__checkbox')
+            .click({force: true});
+
+        cy.matchImageSnapshot('04-input-card-grouped-disabled', {capture: 'viewport'});
```

Comment:
Do we actually need to take screenshot of the whole page ?

It is so time-expensive(